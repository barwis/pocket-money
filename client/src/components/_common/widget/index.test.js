import React from 'react';
import { shallow, render } from 'enzyme';
import '@testing-library/jest-dom';
import Widget from '.';

describe( 'Widget', () => {
	it( 'should render correctly', () => {
		const component = shallow( <Widget /> );
		expect( component ).toMatchSnapshot();
	});

	it( 'should have proper className', () => {
		// expect(container.firstChild.classList.contains('foo')).toBe(true)
		const testClass = 'this is some class';
		const component = render( <Widget className={testClass} /> );
		expect( component.hasClass( testClass ) ).toBe( true );
	});
});
