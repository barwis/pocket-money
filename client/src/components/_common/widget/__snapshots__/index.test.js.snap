// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Widget should render correctly 1`] = `
<div
  className=" widget-wrapper"
>
  <div />
</div>
`;
